reference/env_vars.rst

```diff
@@ -480,6 +482,11 @@ See values behavior:
       doesn't install any package in your system. Useful if you want to keep manual control of these dependencies,
       for example in your development environment.
 
+Although this value is defaulted to ``enabled``, the default value can be configured with the
+``default_mode`` parameter in the :ref:`SystemPackageTool<systempackagetool>` constructor. If this parameter is passed,
+``CONAN_SYSREQUIRES_MODE`` will take this value if the environment variable has not been previously
+defined.
```

Comment:
I would remove this paragraph, the value of envvar `CONAN_SYSREQUIRES_MODE` won't take the value of the argument `default_mode`.